### PR TITLE
[cpu] Add runtime library for CPU kernels

### PR DESCRIPTION
**Pull request by int3**
This will make it simpler to implement device_assert, device_print etc.

I am putting this up so we can build upon it in parallel (I am working on device_assert and @minjang on device_print).

### CODE REVIEW

File: third_party/cpu/backend/driver.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 from triton.runtime.cache import get_cache_manager
 from triton.backends.driver import DriverBase
 from triton.backends.compiler import GPUTarget
 
-dirname = os.getenv("TRITON_SYS_PATH", default="/usr/local")
-include_dir = [os.path.join(dirname, "include")]
-library_dir = [os.path.join(dirname, "lib")]
+_dirname = os.getenv("TRITON_SYS_PATH", default="/usr/local")
+# for locating libTritonCPURuntime
+_triton_C_dir = importlib.resources.files(triton._C).joinpath("")
```
Comment (int3) [inline third_party/cpu/backend/driver.py:14]: going through `importlib` ensures we can locate the library regardless of whether we are pip installing in editable or regular mode

**Comment by ienkovich** [comment]
The patch is missing the runtime itself.

**Comment by int3** [comment]
oops, fixed.

**Review by minjang — APPROVED** [review]
Great to see it!

**Comment by int3** [comment]
ah sorry, I thought the CI tests were consistently timing out

Let me look into that

**Comment by int3** [comment]
Fix: https://github.com/triton-lang/triton-cpu/pull/75

**Pull request merged**